climada_petals/entity/exposures/test/test_openstreetmap.py

```diff
@@ -131,73 +51,71 @@ def test_area_to_queryformat(self):
         """test methods of OSMApiQuery" """
         area_bbox = (8.5327506, 47.368260, 8.5486078, 47.376877)
         area_poly = shapely.geometry.Polygon([(8.5327506, 47.368260), (8.5486078, 47.376877), (8.5486078, 47.39)])
         condition_building = '["building"]'
         condition_church = '["amenity"="place_of_worship"]'
-
+        
         q1 = osm_dl.OSMApiQuery(area_bbox,condition_building)._overpass_query_string()
         q2 = osm_dl.OSMApiQuery(area_poly,condition_church)._overpass_query_string()
-
+        
         self.assertEqual(
             q1, '[out:json][timeout:180];(nwr["building"](47.36826, 8.5327506, 47.376877, 8.5486078);(._;>;););out;')
         self.assertEqual(
             q2, '[out:json][timeout:180];(nwr["amenity"="place_of_worship"](poly:"47.36826 8.5327506 47.376877 8.5486078 47.39 8.5486078 47.36826 8.5327506");(._;>;););out;')
-
+    
     def test_assemble_from_relations(self):
         """test methods of OSMApiQuery" """
         pass
-
+    
     def test_assemble_from_ways(self):
         """test methods of OSMApiQuery" """
         pass
-
+    
     def test_assemble_from_nodes(self):
         """test methods of OSMApiQuery" """
         pass
-
+    
     def test_update_availability(self):
         """test methods of OSMApiQuery" """
         pass
-
+    
     def test_assemble_results(self):
         """test methods of OSMApiQuery" """
         pass
-
+    
     def test_osm_geoms_to_gis(self):
         """test methods of OSMApiQuery" """
         pass
-
+    
     def skip_test_get_data_overpass(self):
-        """test methods of OSMApiQuery"
+        """test methods of OSMApiQuery" 
         skipping: test causes a segfault on Jenkins"""
         area_bbox = (8.5327506, 47.368260, 8.5486078, 47.376877)
         area_poly = shapely.geometry.Polygon([(8.5327506, 47.368260), (8.5486078, 47.376877), (8.5486078, 47.39)])
         condition_building = '["building"]'
         condition_church = '["amenity"="place_of_worship"]'
         gdf1 = osm_dl.OSMApiQuery(area_bbox,condition_building).get_data_overpass()
         gdf2 = osm_dl.OSMApiQuery(area_poly,condition_church).get_data_overpass()
-
+        
         self.assertTrue(len(gdf1)>0)
         self.assertTrue(len(gdf2)>0)
         self.assertTrue('building' in gdf1.iloc[randint(0,len(gdf1)-1)].tags.keys())
         self.assertTrue('amenity' in gdf2.iloc[randint(0,len(gdf2)-1)].tags.keys())
         self.assertTrue('place_of_worship' in gdf2.iloc[randint(0,len(gdf2)-1)].tags['amenity'])
-
+    
 # Execute Tests
 if __name__ == "__main__":
-    TESTS = unittest.TestLoader().loadTestsFromTestCase(TestOSMRaw)
-    TESTS.addTests(unittest.TestLoader().loadTestsFromTestCase(TestOSMFileQuery))
-    TESTS.addTests(unittest.TestLoader().loadTestsFromTestCase(TestOSMApiQuery))
-    unittest.TextTestRunner(verbosity=2).run(TESTS)
+    TESTS = unittest.TestLoader().loadTestsFromTestCase(TestOSMApiQuery)
+    unittest.TextTestRunner(verbosity=2).run(TESTS)
```

Comment:
Missing newline at end of file
```suggestion
    unittest.TextTestRunner(verbosity=2).run(TESTS)

```